jsonpath/0.8.1.wso2v1/pom.xml

```diff
@@ -0,0 +1,120 @@
+    		<artifactId>json-path</artifactId>
+    		<version>0.8.1</version>
+	</dependency>
+	<dependency>
+    		<groupId>com.jayway.jsonpath</groupId>
+    		<artifactId>json-path-assert</artifactId>
+    		<version>0.8.1</version>
+    		<scope>test</scope>
+		<optional>true</optional>
+	</dependency>
+	
+
+	<dependency>
+		<groupId>commons-lang</groupId>
+		<artifactId>commons-lang</artifactId>
+		<version>2.6</version>
+		<optional>true</optional>
+	</dependency>
+            
+            
+ </dependencies>
+	
+   <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>fail</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+	        <version>1.4.0</version>               
+                <extensions>true</extensions>
+		
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+			 com.jayway.jsonpath.spi.impl;uses:="com.jayway.jsonpath,org.codehaus.jackson.map.type,net.minidev.json,net.minidev.json.parser,
+			 org.codehaus.jackson.type,org.apache.commons.lang,org.codehaus.jackson.map,com.jayway.jsonpath.spi,org.codehaus.jackson",
+			 com.jayway.jsonpath.internal.filter;uses:="com.jayway.jsonpath.spi,com.jayway.jsonpath,com.jayway.jsonpath.internal.filter.eval",
+			 com.jayway.jsonpath.internal;uses:="com.jayway.jsonpath.internal.filter,com.jayway.jsonpath.spi,com.jayway.jsonpath",
+			 com.jayway.jsonpath.spi;uses:="com.jayway.jsonpath.spi.impl,com.jayway.jsonpath",
+			 com.jayway.jsonpath;uses:="com.jayway.jsonpath.internal.filter,com.jayway.jsonpath.internal,com.jayway.jsonpath.spi,
+			 org.apache.commons.lang",!com.jayway.jsonpath.internal.*
+			</Export-Package>
+			<Private-Package>
+           			com.jayway.jsonpath.internal.*
+	               	</Private-Package>
+
+                        <Import-Package>
```

Comment:
Better to organize the imports one by one so it will be easy for others to check packages.